client/src/javascript/components/modals/settings-modal/AboutTab.js

```diff
@@ -0,0 +1,27 @@
+            <br/>
+            Flood is another web interface for <a href="https://github.com/rakshasa/rtorrent" rel="noopener noreferrer" target="_blank">rtorrent</a>. It implements a Node.js server for communicating with the rTorrent API, storing historical data, and serving the web UI.
```

Comment:
These strings should use `FormattedMessage` like other strings in the app.

[Here's an example of `FormattedMessage`](https://github.com/jfurrow/flood/blob/master/client/src/javascript/components/modals/settings-modal/ConnectivityTab.js#L40-L43), and [here's the corresponding English copy](https://github.com/jfurrow/flood/blob/master/client/src/javascript/i18n/en.js#L140).

Reply:
How could it be done with links inside ?